### ISSUE TITLE
Jigasi : Transcriber - not a known participant

### ISSUE THREAD

**Issue by jon101101** [jitsi/jigasi#91]
Hi,

I have ran into an issue when the transcriber joins. SIP works, in fact I can call mobile numbers fine. When the transcriber joins it posts the correct meeting welcome message. 

`Started transcription!
Transcript will be available after the conference at
https://localhost/transcript_2017-11-16T11:45:39.751Z_14d98999-5692-45fd-8837-04894d919dcb.txt`

It also appears that the file is created when the transcriber is booted. Although it's empty. However when looking at the jigasi logs this shows:

2017-11-16 12:45:58.543 WARNING: [359] org.jitsi.jigasi.transcription.Transcriber.log() Reading from SSRC 327414458 while it is not known as a participant

Looking at this: https://github.com/jitsi/jigasi/blob/master/src/main/java/org/jitsi/jigasi/transcription/Transcriber.java#L436

it appears this is the offending line, so why is `Participant p = participants.get(ssrc);` returning null? Is it down to how I have configured the jitsi components?

**Comment by bgrozev** [comment]
We also see this problem, but very rarely, so we haven't investigated yet. Do you see it consistently in your setup?

**Comment by jon101101** [comment]
Yes, i've never seen a working transcription. It happens all the time, I tried different conference configurations e.g. 2 real people in there plus the transcriber and it happens for each of them too. 

Is there a way to test the transcription more thoroughly? I don't think it's sending the audio to Google. As my account overview/quota is still at 0.

Thanks

**Comment by bgrozev** [comment]
If you see the message about an unknown participant, then audio is not being sent to Google (for this participant). Note that the Google interface might not update immediately and still show 0 after a while.

**Comment by jon101101** [comment]
Can I force it to send for unknowns? 

Or can I make my participants known somehow?

**Comment by HighwayStar** [comment]
I'm still facing this issue when using latest stable docker build and connecting from Firefox. There is no such issue when using Chrome.

**Comment by HighwayStar** [comment]
> I'm still facing this issue when using latest stable docker build and connecting from Firefox. There is no such issue when using Chrome.

Is it known issue with Firefox, or maybe there is known workaround?
